terraform/gcp/buckets.tf

```
@@ -0,0 +1,8 @@
+resource "google_storage_bucket" "participant_documents" {
+  name     = var.documents_bucket_name
+  location = var.region
+
```

```suggestion
  # no public access allowed
  public_access_prevention    = "enforced"

  # only allow access if you have iam perms
  uniform_bucket_level_access = true
```

Minor nits; I think we should enable [uniform bucket level access](https://cloud.google.com/storage/docs/uniform-bucket-level-access) to simplify perms and it's slightly more secure. Also public access prevention, which just prevents accidentally making IAM roles that make it publicly accessible.